core/domain/topic_fetchers.py

```diff
@@ -622,13 +622,14 @@ def get_canonical_story_dicts(
         pending_node_titles = [node.title for node in pending_nodes]
         completed_node_titles = utils.compute_list_difference(
             story_summary.node_titles, pending_node_titles)
-        # Here, the return type of 'to_human_readable_dict()' method is
-        # HumanReadableStorySummaryDict which does not have topic_url_fragment,
-        # story_is_published and other keys. To overcome this missing keys
-        # issues, we defined a CannonicalStoryDict and assigned it to the
-        # `story_summary_dict`. Due this a conflict in type assignment is
-        # raised which cause MyPy to throw `Incompatible types in assignment`
-        # error. Thus to avoid error, we used ignore here.
+        # Here we use MyPy ignore because the return type of
```

Comment:
There should be a new type defined here, right?

Reply:
I think we are already defining new type `CannonicalStoryDict`